### PR TITLE
`BrandExplained` view

### PR DESCRIPTION
**Pull request by henrikwinston**
Complete content for this view.

### CODE REVIEW

File: components/ui/brand-explained.tsx [hunk 1]
```diff
@@ -88,12 +90,21 @@ const BrandExplained = () => {
               <BrandValueContainer>
                 <BrandValueDescription currentTileProgress={progress}>
                   <h3 className="font-lora font-bold text-white text-center text-[18px] w-[300px]">
```
Comment (imsumoregn) [inline components/ui/brand-explained.tsx:92]: These parts of the description could be turned into separate local components to centralise the UI logic. Either do it now, or debt it.

File: components/ui/brand-explained.tsx [hunk 2]
```diff
@@ -120,9 +131,26 @@ const BrandExplained = () => {
...
                   <h2 className="font-lora font-bold text-white text-center text-[18px] w-[300px]">
-                    Giá trị của sự{' '}
-                    <span className="inline-block">trò chuyện.</span>
+                    &ldquo;Certified journey&rdquo;
                   </h2>
+                  <p className="text-white text-[13px] font-manrope w-3/4 text-center leading-[initial]">
+                    Trò chuyện cực kỳ có ích để chia sẻ và kết nối, nhưng đôi
+                    khi thứ thực sự hữu dụng với bạn sẽ phải chi tiết hơn như
+                    thế.{' '}
+                    <span className="inline-block font-bold">PIF Connect</span>,{' '}
```
Comment (imsumoregn) [inline components/ui/brand-explained.tsx:140]: Especially this inline block thing, it's been used in plenty places, if I remember correctly.

File: components/ui/Link/index.tsx [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
-            <NextLink
-                href={href}
-                replace={replace}
-                scroll={scroll}
-                shallow={shallow}
-                locale={locale}
-                prefetch={prefetch}
-                passHref={passHref}
-                className={className}
-                ref={ref}
```
Comment (imsumoregn) [inline components/ui/Link/index.tsx:55]: Also these - just spread the props.

**Review by imsumoregn — APPROVED** [review]

File: components/ui/Link/index.tsx (hunk 3 above)
Comment (henrikwinston) [reply components/ui/Link/index.tsx:55]: Debted

**Pull request merged**